appimage/private/tool/mkappimage.py

```diff
@@ -9,15 +9,15 @@
 from pathlib import Path
 from typing import Dict, Iterable, List, NamedTuple, Optional, Tuple
 
-import rules_python.python.runfiles.runfiles
+from python.runfiles import runfiles
```

Comment:
This change is actually necessary -- Originally this worked by treating the `rules_python` external folder itself as a Py module, but this doesn't work with bzlmod since the external package names/folders have suffixes and stuff. So this is now "correct".